SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -83,6 +91,20 @@ void USpatialWorkerConnection::ConnectToReceptionist(bool bConnectAsClient)
 		UE_LOG(LogSpatialWorkerConnection, Warning, TEXT("No worker type specified through commandline, defaulting to %s"), *ReceptionistConfig.WorkerType);
 	}
 
+#if WITH_EDITOR
+	const bool bSingleThreadedServer = !bConnectAsClient && (GPlayInEditorID != -1);
```

Comment:
There's a sneaky rarely used code path where `PIEIndex` is 0. This happens if you launch as a listen server with `Run as a single process` turned off. Not saying we should support it, but shouldn't crash either.

Reply:
Oh, nice catch. Will update accordingly.